chandra_aca/star_probs.py

```diff
@@ -497,3 +499,53 @@ def prob_minus_p_acq(mag):
         mag = float(mag)
 
     return mag
+
+
+def guide_count(mags, t_ccd):
+    """
+    Given mags from guide stars and a temperature, calculate a guide star
+    fractional count/metric using signal-to-noise scaled mag thresholds.
+
+    :param mags: mags of guide star catalog stars
+    :param t_ccd: ACA CCD temperature at expected time of observation
+    :returns: fractional count
+    """
+    # The bright limit does not scale.
+    thresh0 = 5.9
+    thresh1 = snr_mag_for_t_ccd(t_ccd, ref_mag=10.0, ref_t_ccd=-10.9)
+    thresh2 = snr_mag_for_t_ccd(t_ccd, ref_mag=10.2, ref_t_ccd=-10.9)
+    thresh3 = snr_mag_for_t_ccd(t_ccd, ref_mag=10.3, ref_t_ccd=-10.9)
+    counts = np.zeros_like(mags)
```

Comment:
Not an issue here in practice, but be cautious of `zeros_like` if you can't guarantee the dtype of the input.  If someone passed in an array of ints, e.g. `mags=[7, 8, 9]` this would make counts be an int array and the lines below will fail.

Reply:
Good point.  I switched this to np.zeros(len(mags)).  There's another zeros_like in the sota model calc but it doesn't seem worth tweaking if we're not using the model.